mithril-aggregator/src/runtime/state_machine.rs

```diff
@@ -193,13 +200,13 @@ impl AggregatorRuntime {
                 } else if let Some(open_message) = self
                     .runner
                     .get_current_non_certified_open_message()
-                    .await?
+                    .await.with_context(|| "AggregatorRuntime can not get the current non certified open message")?
                 {
                     // transition READY > SIGNING
                     info!("→ transitioning to SIGNING");
                     let new_state = self
-                        .transition_from_ready_to_signing(chain_beacon, open_message)
-                        .await?;
+                        .transition_from_ready_to_signing(chain_beacon.clone(), open_message.clone())
+                        .await.with_context(|| format!("AggregatorRuntime can not perform a transition from READY state to SIGNING, new_beacon: '{}', entity_type: '{:?}'", chain_beacon, open_message.signed_entity_type))?;
```

Comment:
This is a bit verbose:
```suggestion
                        .await.with_context(|| format!("AggregatorRuntime can not perform a transition from READY state to SIGNING with entity_type: '{:?}'", open_message.signed_entity_type))?;
```